EIPS/2020 Block Reward Reduction.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: <to be assigned>
+title: 2020 Block Reward Reduction to 0.5 ETH
+author: John Lilic @jlilic twitter.com/johnlilic johnlilicEIP@gmail.com, Jerome de Tychey @jdetychey twitter.com/jdetychey,  Others
+discussions-to: https://ethereum-magicians.org/t/eip-2878-block-reward-reduction-to-0-5-eth/
+status: Draft
+type: Standards Track
+category Core
+created: 2020-08-11
+---
+
+
+## Simple Summary
+
+This EIP will reduce the block reward paid to proof-of-work validators.  Reducing the block reward will maintain the status quo of periodic community-activated block reward reductions.  
+
+## Abstract
+Of the top 4 Proof-of-Work blockchains, Ethereum pays the highest inflation rate for block validation.  Blockchains both larger, and smaller than Ethereum pay lower rates without any adverse effects.  Over-paying for block validators is an inefficiency that negatively affects all ETH holders, as it unnecessarily inflates the montary base and reduces the purchasing power of all Ether holders.   This EIP will reduce the block reward to bring inflation in-line with Bitcoin, which is the largest cryptocurrency by market cap.   Block rewards will thus be adjusted to a base of 0.5ETH, uncle and nephew rewards will be adjusted accordingly.    
+
+## Motivation
+
+A block reward that is too high is inefficient and negatively impacts the ecosystem economically. There is prior precedent for reducing the block reward; it has been done twice in the past in tandem with the diffusion of prior difficulty bombs.  The most recent diffusion, Muir Glacier, did not include a block reward reduction and thus broke the prior status-quo.  This EIP will revive the previous status-quo of periodical block reward reductions based on economic conditions.   With the upcoming release of ETH2.0 Phase 0 staking, inflationary pressures on Ethereum will be further increased.  Reducing the block reward prior to ETH2.0 Phase 0 staking will assist in alleviating negative inflationary effects.
+
+## Specification
+Parameters in the source code that define the block reward will be adjusted accordingly.
```

Comment:
This should specify what the block reward will be and any other changes this EIP requires.

Reply:
It also needs to specify what constants are changing, and what they are changing too.

Reply:
All previous EIPs that lowered the block reward, did not specify which constants are changing, since the change is so trivial.  (only one constant is changed).  Would you like us to change this status quo?  

If so, we will make a reference as following for example: 

consensus.go should be updated on line 45:

NewBlockReward = big.NewInt(5e+17) // Block reward in wei for successfully mining a block upward from the fork that will include this EIP

Please advise if this is sufficient, and if not please be specific as to what other changes to what files you would like to see.

Reply:
@JLilic please see the specification section in [EIP-649](https://eips.ethereum.org/EIPS/eip-649) and [EIP-1234](https://eips.ethereum.org/EIPS/eip-1234) to see how these changes have been defined historically.

Reply:
Got it. We will make a similar change in that case.